libkineto/src/CuptiActivityProfiler.cpp

```diff
@@ -1269,23 +1299,17 @@ const time_point<system_clock> CuptiActivityProfiler::performRunLoopStep(
         VLOG_IF(1, currentIter > 0)
             << "This state change was invoked by application's step() call";
 
-        if (libkineto::api().client()) {
-          libkineto::api().client()->stop();
-        }
-
-#if defined(HAS_CUPTI) || defined(HAS_ROCTRACER)
-        if (cupti_.stopCollection) {
-          ecs_.cupti_stopped_early = cupti_.stopCollection;
-          LOG(ERROR)
```

Comment:
Those lines are added to the main branch after this pr. I'd rebase from master branch, and update this pr. 
![image](https://github.com/user-attachments/assets/c2014929-f3db-410a-8a60-f89126b7e16e)

Reply:
Oh interesting, that diff was supposed to be only formatting hmm. I'll add it myself, thanks!

Reply:
https://github.com/pytorch/kineto/compare/main...staugust:kineto:main2_new , I have reorganized the code on a new branch based on Kineto's main branch, hoping this will help.